lib/auth/__tests__/auth.spec.js

```diff
@@ -96,3 +97,47 @@ test.serial('getSession from cookies', async t => {
 
   fetchMock.restore()
 })
+
+test.serial('getSession redirects to sign-thru when JWT is expired', async t => {
+  const req = { }
+  const store = {
+    dispatch: sinon.fake()
+  }
+
+  const fakeLocationReplace = sinon.fake()
+
+  const realWindow = global.window
+  // mock global window object
+  global.window = {
+    location: {
+      protocol: 'https:',
+      host: 'beta.voluntarily.nz',
+      pathName: '/home',
+      search: '?tab=history',
+      hash: '',
+      replace: fakeLocationReplace
+    }
+  }
```

Comment:
This mock window implementation feels kind of hacky. I'm not sure if theres a better way.

Reply:
we can mock the router class but for a simple test this is clear enough.